### PR TITLE
Trials

### PR DESCRIPTION
**Pull request by Blossom-Aniedi**
Added README and MIT files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,33 +17,17 @@
...
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+- Cloned this reprository
```
Comment (Blossom-Aniedi) [inline README.md:21]: Good instructions for project setup

File: index.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Document</title>
+      <link rel="stylesheet" href="./style.css">
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+  crossorigin="anonymous"
+/>
+</head>
+<body>
+      <nav id="navbar" class="nav">
```
Comment (Blossom-Aniedi) [inline index.html:14]: Great use of HTML semantic tags

File: style.css [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+      display: block;
+      font-size: 2.2rem;
+      padding: 2rem;
+    }
+    
+    .nav-list a:hover {
+      background: var(--main-blue);
+    }
+
+    .welcome-section {
```
Comment (Blossom-Aniedi) [inline style.css:88]: Great implementation of flexbox

**Comment by Blossom-Aniedi** [comment]
@Blossom-Aniedi Good job so far!
There's nothing else to be done just merge pull request🥳🎉🎉
## Highlights
:heavy_check_mark: Good commit messages
:heavy_check_mark: No linter error
:heavy_check_mark: Implementing the requirements
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Use of inline comments.

## Required Changes ♻️
_Check the comments under the review._

- Even though you have used inline comments, the usage is wrong. Github inline comments should be restricted to point out where you want the new change to be introduced or where you are suggesting an improvement. If you want to point out something you find good about the code you are reviewing, you should point that out in the review summary, like this one you are reading now. Can you see the 'highlights' section above, yeah, right there.
-  You can still find some code where you can recommend some improvements. Find a piece of code and use the inline comments to recommend the improvements. If you can't find anything you can improve upon, then go ahead, and approve the PR. But remember, your comments should always come as suggestions and not as orders.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Blossom-Aniedi — COMMENTED** [review]
Hi @Blossom-Aniedi,

Good job so far! I see you put a lot of effort into this project. There is nothing else to say other than it's time to merge it!!

👍 Great implementation of flexbox
👍 great use of semantic tags

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by Reem-lab — APPROVED** [review]
## Hi @Blossom-Aniedi, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
 - good job using inline comments 🔥🔥

## Optional suggestions
[ OPTIONAL ] You can also consider using emojis and GIFs They make the code review more human-friendly ✅✅🍄🍄
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**